src/main/scala/cromwell/engine/StoreActor.scala

```diff
@@ -36,28 +38,26 @@ class StoreActor(namespace: WdlNamespace, inputs: WorkflowCoercedInputs) extends
   private val log = Logging(context.system, this)
 
   override def receive: Receive = LoggingReceive {
-
     case CallCompleted(call, callOutputs) =>
-      sender ! handleCallCompleted(call, callOutputs)
+      updateOutputs(call, callOutputs)
+      val msg = if (executionStore.isWorkflowDone) WorkflowActor.Complete else buildRunnableCalls
+      sender() ! msg
```

Comment:
although it only seems to care when `sender` is on the rhs, which I suppose makes sense